src/schema/v2/artist/__tests__/helpers.test.ts

```diff
@@ -133,4 +133,58 @@ describe("getArtistInsights", () => {
       })
     })
   })
+
+  describe("gaining followers insight", () => {
```

Comment:
**suggestion:** reduce outer loop tests

If we extract the function that computes the follower growth percentage then I would reduce the number of tests here - maybe just for the happy path?

*format via [Conventional Comments](https://conventionalcomments.org/)*